### ISSUE TITLE
"open folder" link in Welcome screen doesn't work in 3.10.1

### ISSUE THREAD

**Issue by bpmct** [cdr/code-server#3436]
## OS/Web Information

- Web Browser: Google Chrome, Firefox
- Local OS: OSX Big Sur
- Remote OS: Tested on OSX Bug Sur, and an Ubuntu container in Coder
- Remote Architecture: AMD64
- `code-server --version`: 3.10.1

## Steps to Reproduce

1. Open code-server 3.10.1
2. Navigate to "Help > Welcome"
3. Click "Open folder" link

## Expected

The open folder dialog should appear, as which happens with File > Open

## Actual

Nothing happens

## Logs

No valid logs from code-server --log debug. However, in Output -> Log (Window) the following error occurs

```
[2021-05-21 08:44:40.643] [window] [error] command 'workbench.action.files.openFileFolder' not found: _tryExecuteCommand@http://localhost:8080/static/421237f499079cf88d68c02163b70e2b476bbb0d/usr/local/Cellar/code-server/3.10.1/libexec/lib/vscode/out/vs/workbench/workbench.web.api.js:1735:3364
executeCommand/<@http://localhost:8080/static/421237f499079cf88d68c02163b70e2b476bbb0d/usr/local/Cellar/code-server/3.10.1/libexec/lib/vscode/out/vs/workbench/workbench.web.api.js:1735:3245
```

Once this is done, replicate the issue you're having then collect logging
information from the following places:

1. The most recent files from ~/.local/share/code-server/coder-logs.
2. The browser console.
      Same error as above

    ```
      ERR command 'workbench.action.files.openFileFolder' not found: Error: command 'workbench.action.files.openFileFolder' not found
        at h._tryExecuteCommand (http://localhost:8080/static/421237f499079cf88d68c02163b70e2b476bbb0d/usr/local/Cellar/code-server/3.10.1/libexec/lib/vscode/out/vs/workbench/workbench.web.api.js:1735:3364)
        at http://localhost:8080/static/421237f499079cf88d68c02163b70e2b476bbb0d/usr/local/Cellar/code-server/3.10.1/libexec/lib/vscode/out/vs/workbench/workbench.web.api.js:1735:3245
        at async u.open (http://localhost:8080/static/421237f499079cf88d68c02163b70e2b476bbb0d/usr/local/Cellar/code-server/3.10.1/libexec/lib/vscode/out/vs/workbench/workbench.web.api.js:628:17651)
        at async a.open (http://localhost:8080/static/421237f499079cf88d68c02163b70e2b476bbb0d/usr/local/Cellar/code-server/3.10.1/libexec/lib/vscode/out/vs/workbench/workbench.web.api.js:628:19675)
    ```

    3. The browser network tab.

Nothing relevant 

This issue can be reproduced in VS Code: No

**Comment by sestegra** [comment]
Duplicate of #3427 and #2921

**Comment by bpmct** [comment]
oh thanks. somehow missed that
